Gruntfile.js

```diff
@@ -60,7 +60,7 @@ module.exports = function (grunt) {
 				requireSpaceBeforeObjectValues: true,
 				validateIndentation: '\t',
 				validateLineBreaks: 'LF',
-				validateQuoteMarks: true,
+				validateQuoteMarks: {mark: '\'', escape: false},
```

Comment:
Oh, I see. Because they encountered "use strict" they consider the whole file should use double quotes. Wouldn't be easier to use `'use strict'` with single quotes then? If we do that then we probably don't need to change the jshint rule.